src/com/android/messaging/datamodel/ParticipantRefresh.java

```diff
@@ -390,8 +390,9 @@ private static void refreshSelfParticipantList() {
             // For subscriptions already in the database, refresh ParticipantColumns.SIM_SLOT_ID.
             for (final Integer subId : activeSubscriptionIdToRecordMap.keySet()) {
                 final SubscriptionInfo record = activeSubscriptionIdToRecordMap.get(subId);
-                final String displayName =
-                        DatabaseUtils.sqlEscapeString(record.getDisplayName().toString());
+                if (record == null) continue;
```

Comment:
This `record == null` check isn't needed since it's created above and the entries in the map can't be `null`.